msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -945,7 +945,7 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
 
 **Optional** &ndash; Object
 
-The `extensions` property specifies Outlook Add-ins within an app manifest and simplify the distribution and acquisition across the Microsoft 365 ecosystem. Each app supports only one extension.
+The `extensions` property specifies Outlook add-ins within an app manifest and simplifies the distribution and acquisition across the Microsoft 365 ecosystem. Each app supports only one extension.
```

Comment:
Just noticed this. We're going to have to change this to Office in a few weeks anyway and Outlook doesn't support the context menus anyway, so let's do this now. 

```suggestion
The `extensions` property specifies Office add-ins within an app manifest and simplifies the distribution and acquisition across the Microsoft 365 ecosystem. Each app supports only one extension.
```